src/test/java/seedu/address/testutil/TestUtil.java

```diff
@@ -36,20 +36,20 @@ public static Path getFilePathInSandboxFolder(String fileName) {
      * Returns the middle index of the person in the {@code model}'s person list.
      */
     public static Index getMidIndex(Model model) {
-        return Index.fromOneBased(model.getFilteredPersonList().size() / 2);
+        return Index.fromOneBased(model.getFilteredParticipantList().size() / 2);
     }
 
     /**
      * Returns the last index of the person in the {@code model}'s person list.
      */
     public static Index getLastIndex(Model model) {
-        return Index.fromOneBased(model.getFilteredPersonList().size());
+        return Index.fromOneBased(model.getFilteredParticipantList().size());
     }
 
     /**
      * Returns the person in the {@code model}'s person list at {@code index}.
      */
     public static Person getPerson(Model model, Index index) {
-        return model.getFilteredPersonList().get(index.getZeroBased());
+        return model.getFilteredParticipantList().get(index.getZeroBased());
     }
```

Comment:
Should this method be changed? It is probably gonna involve quite a bit of change 😓

Reply:
I think he's made the necessary changes in Model/ModelManager already so should be okay

Reply:
done nps at all :)